client/src/views/browse.html

```diff
@@ -0,0 +1,14 @@
+<div class="row">
+    <div class="small-12 text-center">
+        <br>
+        <b>Browse For Siafiles.</b>
+    </div>
+</div>
+
+<div data-filter></div>
+
+<div class="row">
+    <div class="large-8 medium-10 small-12 large-offset-2 medium-offset-1">
+        <table data-siafile-table></table>
+    </div>
+</div>
```

Comment:
Seems weird to me to have so many tiny files, but maybe it's the best way to approach the problem.